File: backend/core/choices.py
# core/choices.py

# ユーザーの役割
ROLE_CHOICES = [
    ('client', '依頼人'),
    ('coordinator', '仲介人'),
    ('worker', '実行人'),
]

# タスク種類
TASK_TYPE_CHOICES = [
    ('software_dev', 'ソフトウェア開発'),
    ('video_edit', '動画編集'),
    ('writing', 'ライティング/コンテンツ作成'),
    ('translation', '翻訳'),
]

# タスクステータス
STATUS_CHOICES = [
    ('open', '公開中'),
    ('in_progress', '進行中'),
    ('completed', '完了'),
    ('canceled', '取り下げ'),
]

# 仲介リクエストステータス
REQUEST_STATUS_CHOICES = [
    ("pending", "処理待ち"),
    ("approved", "承認済み"),
    ("rejected", "拒否"),
]

File: frontend/src/config/choices.js
export const ROLE_CHOICES = [
  { value: 'client', label: '依頼人' },
  { value: 'coordinator', label: '仲介人' },
  { value: 'worker', label: '実行人'},
];

export const TASK_TYPE_CHOICES = [
  { value: 'software_dev', label: 'ソフトウェア開発' },
  { value: 'video_edit', label: '動画編集' },
  { value: 'writing', label: 'ライティング/コンテンツ作成' },
  { value: 'translation', label: '翻訳' },
];

export const STATUS_CHOICES = [
  { value: 'open', label: '公開中' },
  { value: 'in_progress', label: '進行中'},
  { value: 'completed', label: '完了' },
  { value: 'canceled', label: '取り下げ' },
];

export const REQUEST_STATUS_CHOICES = [
  { value: 'pending', label: '処理待ち' },
  { value: 'approved', label: '承認済み' },
  { value: 'rejected', label: '拒否' },
];